contents/handbook/company/structure.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+PostHog is structured for speed, autonomy and innovation.
```

Comment:
Generally on this page it would be good to link back to other pages in the Handbook where we talk about our values, treading on toes etc.

Reply:
I found this tricky to do easily. Feel free to create a new PR if you fancy.